src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -1226,6 +1226,21 @@ using RemoveTags = tmpl::flatten<tmpl::list<Tags...>>;
 template <typename... Tags>
 using AddSimpleTags = tmpl::flatten<tmpl::list<Tags...>>;
 
+/*!
+ * \ingroup DataBoxGroup
+ * \brief List of Tags to mutate in the DataBox
+ */
+template <typename... Tags>
+using MutateTags = tmpl::flatten<tmpl::list<Tags...>>;
+
+/*!
+ * \ingroup DataBoxGroup
+ * \brief List of Tags in addition to MutateTags nneeded from the DataBox
```

Comment:
- typo `nneeded`
- Perhaps I misunderstand, but wouldn't this be more clearly documented as "List of Tags to get from the DataBox to be used as arguments", or similar?